docs/how-to-guides/how-to-setup-triggers.md

```diff
@@ -0,0 +1,101 @@
+To find a complete list of predefined variables, supported operations, and configuration details, look at triggers [reference guide](https://memgraph.com/docs/memgraph/reference-guide/triggers).
+
+In this trigger via `SET createdVertex.created = timestamp()`  command, new node is getting a timestamp after creation.
+
+### Trigger execution upon node update
+
+Node property updates are common in graphs,in order to react to them you can create trigger for that type of event: 
+
+```cypher
+CREATE TRIGGER updateVertex
+ON () UPDATE AFTER COMMIT EXECUTE
+UNWIND updatedVertices AS updatedVertex
+SET updatedVertex.updated_at = timestamp()
+
+```
+The trigger for node updates is almost the same as node creation. Notice different predefined variable `updatedVertices` and `ON UPDATE` cypher command.
+
+In this trigger via `SET updatedVertex.updated_at = timestamp()` command, node is getting new updated timestamp.
+
+### Trigger execution upon node or relationship creation
+
+You can also setup the trigger for multiple events, such as node or relationships creation. It does matter what will be created node or relationship, but it will execute the trigger.
+Here is the sample query for the trigger: 
+
+```cypher
+CREATE TRIGGER exampleTrigger
+ON CREATE AFTER COMMIT EXECUTE
+UNWIND createdObjects AS createdObject
+WITH CASE
+        WHEN createdObject.vertex IS NOT null THEN createdObject.vertex
+        WHEN createdObject.edge IS NOT null THEN createdObject.edge
+    END AS object
+SET object.created_at = timestamp();
+```
+
+By using predefined variable `createdObjects` you are getting all created objects. Variable `createdObjects` is a list of maps where each map contains `event_type` and `edge` or `vertex` property. Both properties depends on 
+what type of object has been created. If your Query is creating multiple nodes and relationships, they will be in a separate created objects maps. 
+
+In this trigger via `SET object.created_at = timestamp();` command, node or relationships property `created_at` is set to current timestamp. 
+
+## How to create trigger for Python query module? 
+
+If you what to react on event by executing code from query module in Python, you can do it by calling query module from trigger. Sample trigger below will call `query_module.new_edge(edge)` on each new relationship (edge) creation:
+
+```cypher
+CREATE TRIGGER newEdge
+ON CREATE BEFORE COMMIT EXECUTE
+UNWIND createdEdges AS edge
+CALL query_module.new_edge(edge) YIELD *;
+```
+
+Make sure that your function accepts proper Memgraph type, `mgp.Edge` in this case. 
+
+```python
+def new_edge(
+    context: mgp.ProcCtx,
+    edge: mgp.Edge
+)
+```
+Memgraph Python API is defined via `mgp.py` script an there you can find all supported types and  types such as `mgp.Edge`, `mgp.Vertex` etc. If you what to explore the API further, feel free to check the reference guide on [Python API](https://memgraph.com/docs/memgraph/reference-guide/query-modules/api/python-api)
```

Comment:
....defined BY mgp.py script and in it you can find all supported types, such as bla, bla
There is no dot after the second sentence